cmd/cosign/cli/attest/attest.go

```diff
@@ -86,6 +87,9 @@ func (c *AttestCommand) Exec(ctx context.Context, imageRef string) error {
 	if err != nil {
 		return err
 	}
+	if c.RegistryOptions.AllowHTTPRegistry || c.RegistryOptions.AllowInsecure {
+		ociremoteOpts = append(ociremoteOpts, ociremote.WithNameOptions(name.Insecure))
```

Comment:
Minor: so our `RegistryOptions.AllowHTTPRegistry` maps to `name.Insecure` which both mean "it's okay if the registry uses plain HTTP without TLS.

But my interpretation of `RegistryOptions.AllowInsecure` is that the registry is using TLS, but with a self-signed certificate (or at least a certificate that doesn't verify with the machine's certificate store).

🤷 Are there people actually using that configuration? Or is everyone just using an HTTP registry? I guess we'll find out! I didn't see a "use TLS but allow certificates that don't verify with the machine's certificate store" on https://pkg.go.dev/github.com/google/go-containerregistry@v0.20.6/pkg/name.

Reply:
Yeah, it didn't seem like go-containerregistry makes a distinction between them.